.circleci/config.yml

```
@@ -18,13 +18,14 @@ jobs:
     docker:
       - image: circleci/buildpack-deps:18.04
         environment:
-          # NOTE(kan-bayashi): Use 3.7.3 to avoid sentencepiece installation error
-          ESPNET_PYTHON_VERSION: 3.7.3
+          ESPNET_PYTHON_VERSION: 3.6
           TH_VERSION: 1.1.0
           CHAINER_VERSION: 5.0.0
```

In `setup.py` we use chainer==6.0.0.
Maybe it is better to follow it.
```suggestion
          CHAINER_VERSION: 6.0.0
```